dataactbroker/handlers/accountHandler.py

```diff
@@ -366,6 +432,27 @@ def changeStatus(self,system_email):
         self.interfaces.userDb.changeStatus(user,requestDict.getValue("new_status"))
         return JsonResponse.create(StatusCode.OK,{"message":"Status change successful"})
 
+    def listUsers(self):
+        """ List all users ordered by status. Associated request body must have key 'filter_by' """
+        user = self.interfaces.userDb.getUserByUID(LoginSession.getName(flaskSession))
+        isAgencyAdmin = True if self.interfaces.userDb.hasPermission(user, "agency_admin") else False
+        try:
+            if isAgencyAdmin:
+                users = self.interfaces.userDb.getUsers(agency=user.agency)
+            else:
+                users = self.interfaces.userDb.getUsers()
```

Comment:
I'm confused by the purpose here.  What is the set of users someone who is not an admin is allowed to see?  If this is empty or just themselves, shouldn't we just require agency_admin in order to access the route?